index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+     <link rel="stylesheet" href="styles.css"> 
+</head>
+<body>
+  <header class="tool-bar">
+    <div class="toolbar-dark-compact">
+        <a class="john-doe" href="https://www.w3schools.com">Yasen</a>
+        <nav class="normal-button" >  
+            <img  src="img/Union.png" alt="">
+        </nav>
+    </div>
+  </header>
+  <section class="head-line">
+    <h1 class="primary-text">
+        Hey There. I’m Yaseen
+    </h1>
+    <h2 class="supporting-text">
+        I’am a Software Developer
+    </h2>
+    <p class="discription">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate 
+        <br>to contact me.
+    </p>
+    <ul class="social-media">
+        <li  class="social-media-buttons">
+            <a href="https://github.com/">
+                <img class="social-media-icons cat-icon" src="img/cat.svg" alt="">
+            </a>
+        </li>
+        <li  class="social-media-buttons">
+            <a href="https://www.linkedin.com/">
+                <img class="social-media-icons" src="img/in.png" alt="">
+            </a>
+            </li>
+        <li  class="social-media-buttons">
+            <a href="">
+                <img class="social-media-icons" src="img/hand.png" alt="">
+            </a>
+        </li>
+        <li  class="social-media-buttons">
+            <a href="https://twitter.com">
+                <img class="social-media-icons" src="img/bird.png" alt="">
+            </a>
+        </li>
+        <li  class="social-media-buttons">
+            <a href="">
+                <img class="social-media-icons" src="img/m.png" alt="">
+            </a>
+        </li>
+    </ul>
+    <!-- <div class="vectors">
+        <img class="vector" src="img/Illustration.svg" alt="">
+    </div> -->
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT**: Kindly make sure you remove the commented code lines to make your codebase more readable and professional. I know you can do it :muscle: :muscle: